docs/user_guides/fs/transformation_functions.md

```diff
@@ -0,0 +1,163 @@
+
+To create a one-to-one transformation function, the Hopsworks `@udf` decorator must be provided with the `return_type` as a single Python type. The transformation function should take one argument as input and return a Pandas Series.
+
+=== "Python"
+
+    !!! example "Creation of a one-to-one transformation function in Hopsworks."
+        ```python
+        from hopsworks import udf
+
+        @udf(return_type=int)
+        def add_one(feature):
+            return feature + 1
+        ```
+
+### Many-to-one transformations
+
+The creation of many-to-one transformation functions is similar to that of a one-to-one transformation function, the only difference being that the transformation function accepts multiple features as input.
+
+=== "Python"
+    !!! example "Creation of a many-to-one transformation function in Hopsworks."
+        ```python
+        from hopsworks import udf
+
+        @udf(return_type=int)
+        def add_features(feature1, feature2, feature3):
+            return feature + feature2 + feature3
+        ```
+
+### One-to-many transformations
+
+To create a one-to-many transformation function, the Hopsworks `@udf` decorator must be provided with the `return_type` as a list of Python types, and the transformation function should take one argument as input and return multiple features as a Pandas DataFrame. The return types provided to the decorator must match the types of each column in the returned Pandas DataFrame.
+
+=== "Python"
+    !!! example "Creation of a one-to-many transformation function in Hopsworks."
+        ```python
+        from hopsworks import udf
+        import pandas as pd
+
+        @udf(return_type=[int, int])
+        def add_one_and_two(feature1):
+            return pd.DataFrame({"add_one":feature1 + 1, "add_two":feature1 + 2})
+        ```
+
+### Many-to-many transformations
+
+The creation of a many-to-many transformation function is similar to that of a one-to-many transformation function, the only difference being that the transformation function accepts multiple features as input.
+
+=== "Python"    
+    !!! example "Creation of a many-to-many transformation function in Hopsworks."
+        ```python
+        from hopsworks import udf
+        import pandas as pd
+
+        @udf(return_type=[int, int, int])
+        def add_one_multiple(feature1, feature2, feature3):
+            return pd.DataFrame({"add_one_feature1":feature1 + 1, "add_one_feature2":feature2 + 1, "add_one_feature3":feature3 + 1})
+        ```
+
+### Dropping input features
+
+The `drop` parameter of the `@udf` decorator is used to drop specific features after transformation.  If any argument of the transformation function is passed to the `drop` parameter, then the feature mapped to the argument is dropped after the transformation functions are applied. In the example below, the features mapped to the arguments `feature1` and `feature2` are dropped after the application of all transformation functions.
```

Comment:
to drop specific columns in the input DataFrame after transformation

Reply:
Should we use column or feature here?
I think when we refer to DataFrames, we should use the term column.